tensorrt_llm/_torch/models/modeling_minicpmv.py

```
@@ -0,0 +1,1169 @@
+                                  trust_remote_code: bool) -> Any:
+        processor_tokenizer = getattr(self._processor, "tokenizer", None)
+        if self._has_minicpmv_tokenizer_attrs(processor_tokenizer):
+            return processor_tokenizer
+
+        try:
+            processor_tokenizer = AutoTokenizer.from_pretrained(
+                model_path,
+                trust_remote_code=trust_remote_code,
+                use_fast=self.use_fast,
+            )
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)